bip-0360.mediawiki

```diff
@@ -1,724 +1,452 @@
-public key is exposed on the blockchain when the transaction is spent, making it vulnerable to quantum attack until
-it's mined. One way to mitigate this is to submit the transaction directly to a mining pool, bypassing the mempool.
-This process is known as an out-of-band transaction or a private mempool. In this case, the mining pool must be trusted
-not to reveal the transaction public key to attackers. The problem with this approach is that it requires a trusted
-third party, which the P2QRH proposal aims to avoid. It also doesn't account for block reorg attacks, which would
-reveal public keys in blocks that were once mined but are now orphaned and must be mined again. Additionally,
-it depends on the mining pool whether they reveal their block template to either the public or to miners.
-
-Not having public keys exposed on-chain is an important step for quantum security. Otherwise, funds would need to be
-spent to new addresses on a regular basis in order to prevent the possibility of a "long-exposure CRQC attack" recovering
-the key behind high-value addresses. A long-exposure quantum attack can be considered one performed with chain data, such
-as that from a used address or one encoded in a spend script. This is likely to be more common early on, as early
-quantum computers must be run for longer in order to overcome errors caused by noise. A "short-exposure quantum attack"
-would be one performed on keys in the mempool, which is seen as much more difficult given the block time, and so it
-requires more sophisticated CRQCs.<ref name="short-exposure">
-In the paper
-[https://arxiv.org/pdf/2306.08585 How to compute a 256-bit elliptic curve private key with only 50 million Toffoli gates]
-the authors estimate that a CRQC with 28 million superconducting physical qubits would take 8.3 seconds to calculate a
-256-bit key, while a CRQC with 6.9 million physical qubits would take 58 seconds. This implies that a CRQC with 4x as
-many qubits would be roughly 7 times faster.
-</ref>
-
-As the value being sent increases, so too should the fee in order to commit the transaction to the chain as soon as
-possible. Once the transaction is mined, it makes useless the public key revealed by spending a UTXO, so long as it is
-never reused.
-
-As the first step to address these issues we propose Pay to Quantum Resistant Hash (P2QRH), an output type that allows 
-tapscript to be used in a quantum resistant manner.
-This new output type protects transactions submitted to the mempool and helps preserve the fee market by
-preventing the need for private, out-of-band mempool transactions.
-
-The following table is intended to inform the average Bitcoin user whether their Bitcoin is vulnerable to a long-exposure
-quantum attack:
+Given these concerns, we think it’s worth considering changes that are minimal in complexity and risk – and create new options for using Bitcoin in a quantum-resistant way.
```

Comment:
Why are we break each sentence into a separate paragraph?

Reply:
Question for the authors. Will combine some of these where it makes sense.

Reply:
What are you asking here? This seems like a formatting issue where each sentence formatted as if it a new paragraph. I think we should combine them into paragraphs